docs/intro/tutorial.rst

```diff
@@ -160,38 +178,34 @@ returning the result of the ``to_item`` method call. We could make use of the
             links = response.css('.image_container a')
             yield from response.follow_all(links, self.parse_book)
 
+
 .. note::
 
-    You can also write something like ``Request(url, callback_for(BookPage))``,
-    without creating an attribute, but currently it won't work with Scrapy disk
-    queues.
+    You can also write something like
+    ``response.follow_all(links, callback_for(BookPage))``, without creating
+    an attribute, but currently it won't work with Scrapy disk queues.
 
 Final result
 ============
 
-At the end of our job, our spider should look like this:
+At the end of our job, the spider should look like this:
 
 .. code-block:: python
 
     import scrapy
-
-    from scrapy_poet import callback_for
```

Comment:
I think imports are supposed to follow alphabetical order.

Reply:
Why? :) 

Let me explain the logic for the current imports. The rule in pep8 is the following:

> Imports should be grouped in the following order:
>
>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.
>
> You should put a blank line between each group of imports.

Here, technically, all imports are "related third party imports", so pep8 doesn't specify any order.

I ordered them by "relatedness" - scrapy is almost considered stdlib here, and scrapy-poet is almost a local application/library specific import, as we're explaining how scrapy-poet works in the tutorial. I can admit this order is quite arbitrary; is it worse than other orders? I've seen people are ordering imports alphabetically, by length, etc.

Reply:
Sorry, I didn't mean there was a PEP or something.

I'm used to seeing alphabetical order across some Python projects and I like this approach because it's a concrete convention that's easy to be followed by any developer working on the project. Whereas "relatedness" is somewhat subjective to a certain point.

Also, I would hate to nitpick to this point. This was more like a suggestion. Up to you :)

Reply:
Regarding alphabetical order - nothing against it, though for the guidelines which can be automated tooling like https://github.com/timothycrosley/isort may work better; you don't need to spend human brain cycles on something a machine can check & fix :)